stroeer/core/v1/core_curation_service.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+/**
+ * @FileArticle ⚙︎ CurationService
+ */
+
+package stroeer.core.v1;
+
+import "stroeer/core/v1/article.proto";
+
+option go_package = "github.com/stroeer/go-tapir/core/v1;core";
+option java_multiple_files = true;
+option java_package = "de.stroeer.core.v1";
+
+/**
+* ```protobuf
+* service ArticleService {
+*   // resolve a curation and return the `articles` within this curation
+*   rpc GetCuration(GetCurationRequest) returns (GetCurationResponse) {}
+*   //
+*   rpc BatchGetCuration(BatchGetCurationRequest) returns (BatchGetCurationResponse) {}
+* }
+* ```
+*/
+service CurationService {
+  rpc GetCuration(GetCurationRequest) returns (GetCurationResponse) {}
+  rpc BatchGetCuration(BatchGetCurationRequest) returns (BatchGetCurationResponse) {}
+}
+
+/**
+* # `⚙︎ GetCuration`
+* ```protobuf
+*   rpc GetCuration(GetCurationRequest) returns (GetCurationResponse) {}
+* ```
+*
+* Fetch a curation by its id and return the [`repeated stroeer.core.v1.Article`](Article.html) this
+* curation contains. The response may be empty in case the curation does not contain any items.
+*
+* a `NOT_FOUND` status code will indicate the curation `id` does not exist.
+*/
+message GetCurationRequest {
+  int64 id = 1;
+}
+
+message GetCurationResponse {
+  repeated stroeer.core.v1.Article articles = 1;
```

Comment:
we could add the `update_time` of the curation list here as additional info

Reply:
jep, added all the things (label, time, id) to make it more accessible by consumers